scapy/contrib/gtp.uts

```diff
@@ -11,7 +11,7 @@ gtp.dport == 2123 and gtp.teid == 2807 and len(gtp.IE_list) == 5
 
 = GTPCreatePDPContextRequest(), basic dissection
 random.seed(0x2807)
-str(gtp) == b"E\x00\x00O\x00\x01\x00\x00@\x11|\x9b\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x00;{N2\x10\x00+\x00\x00\n\xf7\xd2y\x00\x00\x10\xf8>\x14\x05\x14\t\x85\x00\x04\xa6A\xd8+\x85\x00\x04z\xafnt\x87\x00\x0fxKbPaePK9oq0pb5"
+str(gtp) == b'E\x00\x00O\x00\x01\x00\x00@\x11|\x9b\x7f\x00\x00\x01\x7f\x00\x00\x01\x08K\x08K\x00;\x97\xbd2\x10\x00+\x00\x00\n\xf7\xd2y\x00\x00\x10\xdeM\xb8\xf5\x14\x0f\x85\x00\x04\xabyk\xc1\x85\x00\x04\xb8\xcf\x96\xfe\x87\x00\x0f9Co27Fbj65eKHyQ'
```

Comment:
A RandShort() is `consumed` while the Packet fields are cached. Therefore, the _randomly_ built packet is slightly different after the patch.